### ISSUE TITLE
Daterpicker Accessibility

### ISSUE THREAD

**Issue by prsaroha** [momentum-design/momentum-ui#428]
Currently, Datepicker component does not work with screen reader, since the focus is never actually in the component, and instead CSS is used to highlight elements in the calendar.

## Issue Type
Bug Report

## Component or Pattern Affected
Datepicker

## Expected Behavior
Datepicker should be actually accessible via the keyboard i.e. the components in Datepicker should receive focus and should be the active element, only then will the screen reader read for that element. Arrow keys should be able to navigate the calendar, and tab/shift tab should navigate to and from the next/previous month buttons.
[MeetingsPanelP0_7.pdf](https://github.com/momentum-design/momentum-ui/files/3735313/MeetingsPanelP0_7.pdf)

## Current Behavior
Currently, CSS is used to highlight dates to mimic focus, but since they are not the active elements, the screen reader does not read for those elements. Pressing arrow keys after opening the calendar end up closing the calendar.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Library: React
* Version used: 20.3.0
* Browser Name and version: Chrome 77, Firefox 69
* Operating System and version: Windows 10
* Link to your project:
